src/main/java/duke/tasks/TasksConstant.java

```diff
@@ -0,0 +1,13 @@
+package duke.tasks;
+
+/**
+ * A class for any constants to be used possibly often in its scope.
+ */
+public class TasksConstant {
```

Comment:
I'm just curious why did you use classes to define constants instead of Enums.

Reply:
Hi Cheyanne, it's a personal preference as I foresee different kinds of constants being made and do not want to make too many enums. Creating a constants file would help in the sense everything is stored here, so they can easily be obtained from the TasksConstant.